dothething/dothething/ClipsView.swift

```diff
@@ -35,71 +34,101 @@ struct ClipsView: View {
     @Binding var toggle: Bool
     
     var body: some View {
-        VStack {
-            HStack {
-                // back button
-                Button(action: {
-                    print("Back button tapped")
-                    clipsViewModel.backButtonPressed()
-                    toggle.toggle()
-                }) {
-                    Text("Back")
-                }                                                                                                                                                                                                                                                                                                                                                                                                                                   
-                .padding(.leading)
-                .frame(maxWidth: .infinity, alignment: .leading)
+        GeometryReader { geometry in
+            VStack {
+                ZStack {
+                    HStack {
+                        Button(action: {
+                            print("Back button tapped")
+                            clipsViewModel.backButtonPressed()
+                            toggle.toggle()
+                        }) {
+                            Image(systemName: "chevron.left")
+                                .font(.system(size: 24))
+                                .foregroundColor(.accentColor)
+                        }
+                        .padding(.leading)
+                        Spacer()
+                    }
+                    
+                    Text("domino")
+                        .font(.custom("Montserrat-Medium", size: 27))
+                        .foregroundColor(Color.accentColor)
+                        .tracking(8)
+                        .multilineTextAlignment(.center)
+                }
```

Comment:
The new code introduced here replaces the back button with a more visually appealing implementation. It uses a ZStack to create a navigation bar-like structure, with a chevron-left icon as the back button and the app's title ("domino") centered.

This change improves the overall UI design and provides a better user experience for navigating back to the previous view. The code is well-structured and follows good SwiftUI practices.